src/charm.py

```diff
@@ -116,6 +109,31 @@ def _configure_pebble_layer(
         self._container.replan()
         self.unit.status = ActiveStatus()
 
+    def _on_fiveg_nrf_relation_joined(self, event: RelationJoinedEvent) -> None:
+        """Handle fiveg-nrf relation joined event.
+
+        Args:
+            event: RelationJoinedEvent
+        """
+        if self.unit.is_leader():
```

Comment:
Let's follow the pattern we use everywhere else.

```pyhton
if not self.unit.is_leader():
    return
do_what_you_must()
```

Reply:
✅